### ISSUE TITLE
broken hdmi config parameters on recalbox 8

### ISSUE THREAD

**Issue by St3ffn** [Cacodaimon/CacosGPiCASE2Script#1]
Hi there, 
first of all thank you for your efford in rewriting the scripts from RetroFlag. 

As mentioned by https://github.com/RetroFlag/GPiCase2-Script/issues/6#issuecomment-1017815573 in https://github.com/RetroFlag/GPiCase2-Script/issues/6. On recalbox 8 the two parameters `hdmi_group` and `hdmi_mode` made it impossible  for me to get hdmi output working while using the dock. I fixed it by deleting the two parameters from [config_hdmi.txt](https://github.com/Cacodaimon/CacosGPiCASE2Script/blob/main/configs/config_hdmi.txt). Sadly I don't know if it's a problem for others which are using RetroPie 4.7.1, Recalbox7.1.1 or batocera31. Do you could add that. I am pretty sure that will make Recalbox 8 users super happy :D

A PR in the corresponding RetroFlag repo was also created. See https://github.com/RetroFlag/GPiCase2-Script/pull/14

**Comment by Cacodaimon** [comment]
Hello and thanks for the feedback.

I am using version 8.0.2-Electron and did not face your HDMI issue.

Is it possible that your monitor/TV does not support [1920x1200 60Hz](https://elinux.org/RPi_Configuration) which `hdmi_mode=69` stands for?

But I will try if removing both the parameters does have a negative effect, and if it does not, I will release a new version without the parameters.

**Comment by St3ffn** [comment]
Hi there, thanks for your response. 

I used several monitors/TVs and none of them could display the given mode. Even after settings its default resolution via `hdmi_group` and `hdmi_mode`.  Maybe it's a good starting point to leave those parameters out and point users to the corresponding table. When somebody wants to have a specific hdmi resolution and mode they could still change it manually.

**Comment by Cacodaimon** [comment]
Hello,

I removed `hdmi_mode` etc. from the config files and it works on my TV, too. Now I am facing some over scan issues which can be easily [solved](https://github.com/Cacodaimon/CacosGPiCASE2Script#hdmi-settings) by configuring the HDMI mode there.

After reading the [mode list](https://elinux.org/RPi_Configuration) several times I figured out `hdmi_mode=69` is not Full HD (1920x1080) it is 1920×1200 which is a quite uncommon and larger resolution which could be the reason for your display issues, I have no Idea why Retroflag provides such a config file 🙃.
